app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1> Mechanic <%= @mechanic.id %> </h1>
+<p> Name: <%= @mechanic.name %> </p>
+<p> Years of Experience: <%= @mechanic.years_experience %> </p>
+<p> Rides: 
+  <% @mechanic.rides.each do |ride| %>
+    <%= ride.name %>
+    <br>
+  <% end %>
+</p>
+
+<h3> Add a ride to Mechanic: </h3>
+<%= form_with url: mechanic_rides_path, method: :post, local: true do |form| %>
+  <p> <%= form.label :ride_id %>
+  <%= form.number_field :ride_id %> </p>
+  <p> <%= form.submit "Submit" %> </p>
+  <%= hidden_field_tag :mechanic_id, @mechanic.id %> 
```

Comment:
yeah, this form looks good. Sending the mechanic id through a hidden field works. Not sure why you're using a `hidden_field_tag` instead of a `form.hidden_field` though. Alternatively, you can send additional params through the form via line 12.